.coverage

```diff
@@ -1 +1 @@
-!coverage.py: This is a private format, don't read it directly!{"lines":{"/home/jmason/projects/github/public-cloud-info-client/lib/susepubliccloudinfoclient/__init__.py":[1],"/home/jmason/projects/github/public-cloud-info-client/lib/susepubliccloudinfoclient/infoserverrequests.py":[21,22,23,24,25,26,29,51,66,79,95,110,126,149,168,173,179,204,209,250,254,274,278,283,301,320,251,255,256,257,258,259,260,264,265,266,205,206,267,268,269,270,271,56,58,60,61,63,69,71,73,74,76,116,118,120,121,123,132,134,136,137,139,213,215,217,219,221,223,225,227,229,232,234,235,238,239,240,241,242,247,32,33,34,35,36,37,40,42,43,44,45,46,48,85,87,89,90,92,100,102,104,105,107,245,275,279,280,181,182,184,186,188,192,193,194,195,170,175,151,152,153,154,155,156,157,159,160,161,163,164,165,162,158],"/home/jmason/projects/github/public-cloud-info-client/lib/susepubliccloudinfoclient/version.py":[]}}
\ No newline at end of file
+!coverage.py: This is a private format, don't read it directly!{"lines":{"/work/rjschwei/devel/public-cloud-info-client/lib/susepubliccloudinfoclient/__init__.py":[1],"/work/rjschwei/devel/public-cloud-info-client/lib/susepubliccloudinfoclient/infoserverrequests.py":[21,22,23,24,25,26,29,51,66,79,95,110,126,142,174,179,185,210,219,260,264,284,288,293,306,325,344,363,382,401,261,265,266,267,268,269,270,274,275,276,212,213,214,211,216,277,278,279,280,281,56,58,60,61,63,69,71,73,74,76,116,118,120,121,123,132,134,136,137,139,223,225,227,229,231,233,235,237,239,221,242,244,245,248,249,250,251,252,257,32,33,34,35,36,37,31,40,42,43,44,45,46,48,85,87,89,90,92,100,102,104,105,107,255,285,289,290,187,188,190,192,194,198,199,200,201,176,181,151,152,153,154,155,156,157,158,160,162,165,166,167,169,170,171,168,159,161,163],"/work/rjschwei/devel/public-cloud-info-client/lib/susepubliccloudinfoclient/version.py":[]}}
```

Comment:
Unrelated to the PR but seems like this should be part of gitignore.

Reply:
Agreed